packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -324,8 +325,10 @@ export class VSXExtension implements VSXExtensionData, TreeElement {
             if (plugin) {
                 await this.progressService.withProgress(
                     nls.localizeByDefault('Uninstalling {0}...', this.id), 'extensions',
-                    () => this.pluginServer.uninstall(PluginIdentifiers.componentsToVersionedId(plugin.metadata.model))
-                );
+                    async () => {
```

Comment:
Why go through the command here? Why stringify/destringify the id?

Reply:
> Why go through the command here? Why stringify/destringify the id?

The implementation is moved to a command to resolve #13796 , now calling the new command to reduce code duplicates.